src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println(" run Bulldozer ");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println(" stop Bulldozer ");
+    }
```

Comment:
```suggestion
    public void doWork() {
        System.out.println("The bulldozer started working");
    }

    @Override
    public void stopWork() {
        System.out.println("Bulldozer finished the work");
    }
```
Change the message in all classes